network/endpoint.go

```diff
@@ -53,21 +53,26 @@ type endpoint struct {
 	PODName                  string `json:",omitempty"`
 	PODNameSpace             string `json:",omitempty"`
 	InfraVnetAddressSpace    string `json:",omitempty"`
-	NetNs                    string `json:",omitempty"`
+	NetNs                    string `json:",omitempty"` // used in windows
 	// SecondaryInterfaces is a map of interface name to InterfaceInfo
 	SecondaryInterfaces map[string]*InterfaceInfo
+	// Store nic type since we no longer populate SecondaryInterfaces
+	NICType cns.NICType
```

Comment:
We still have secondaryInterfaces populated on line 58 and each interface object contains NICType field, why we store NICType and NICName here again?

Reply:
we shouldn't use secondary interfaces after this pr goes through (the pr should still support deletion etc. if secondary interfaces are populated though)